### PR TITLE
[Fix]: Hyperlock tvl-ish

### PR DESCRIPTION
**Pull request by phijfry**
Updating an existing file that is reporting TVL incorrectly to make it less incorrect 2M -> 12M -> 17M (wen)

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/hyperlock exports TVL: 
        
 
 
    blast                     12.30 M
    
    total                    12.30 M

File: projects/hyperlock/index.js [hunk 1]
```diff
@@ -1,9 +1,33 @@
...
+  });
+
+  api.add([usdbWethV2], v2Bals);
+
+  await sumTokens2({
+    api,
+    resolveUniV3: true,
+    resolveLP: true,
+    unwrapAll: true,
+    resolveLP: true,
```
Comment (waynebruce0x) [inline projects/hyperlock/index.js:22]: hey can you check this please

**Comment by llamatester** [comment]
The adapter at projects/hyperlock exports TVL: 
        
 
 
    blast                     12.68 M
    
    total                    12.68 M

**Comment by llamatester** [comment]
The adapter at projects/hyperlock exports TVL: 
        
 
 
    blast                     12.69 M
    
    total                    12.69 M

**Pull request merged**